index.html

```diff
@@ -456,11 +456,13 @@ <h2 id="collections">Collection Functions (Arrays or Objects)</h2>
         <b class="header">each</b><code>_.each(list, iteratee, [context])</code>
         <span class="alias">Alias: <b>forEach</b></span>
         <br />
-        Iterates over a <b>list</b> of elements, yielding each in turn to an <b>iteratee</b>
-        function. The <b>iteratee</b> is bound to the <b>context</b> object, if one is
+        Iterates over a <b>list</b> of elements, yielding each in turn to an
+        <b>iteratee</b> function.
+        The <b>iteratee</b> is bound to the <b>context</b> object, if one is
         passed. Each invocation of <b>iteratee</b> is called with three arguments:
-        <tt>(element, index, list)</tt>. If <b>list</b> is a JavaScript object, <b>iteratee</b>'s
-        arguments will be <tt>(value, key, list)</tt>. Returns the <b>list</b> for chaining.
+        <tt>(element, index, list)</tt>. If <b>list</b> is a JavaScript object,
+        <b>iteratee</b>'s arguments will be <tt>(value, key, list)</tt>.
+        Returns the <b>list</b> for chaining.
```

Comment:
Yap, will do. Also, I've totally forgot about this PR, will try to update it with current master once I'll had some spare time.